### ISSUE TITLE
0000904:
finish js frontend for setup

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#543]
**Reported by pschuele on 27 Feb 2009 16:42**

finish js frontend for setup

**Comment by Gloirin** [comment]
Related to #524

**Comment by Gloirin** [comment]
Depends on #544

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 12 Mar 2009 15:49**

todo when finished with children
- removed mode=beta
- add import data (adb, accounts, ...) from egw

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 13 Mar 2009 17:25**

created new task for importing data ....
